docs/git-comparison.md

```diff
@@ -0,0 +1,165 @@
+# Comparison with Git
+
+TODO: Describe the differences compared to Git here
+
+## Command equivalence table
+
+Note that all `jj` commands can be run on any commit (not just the working copy
+commit), but that's left out of the table to keep it simple. For example,
+`jj squash -r <revision>` will move the diff from that revision into its parent.
+
+<table>
```

Comment:
I'd rather use Markdown tables. Googling that gets me https://www.tablesgenerator.com/markdown_tables that might help.

Reply:
I agree with you in principle, but my IDE (IntelliJ) does't seem to know how to format Markdown tables, and I don't want to have to manually align the cell dividers (`|`). HTML is easier to manually format (IntelliJ doesn't seem to format HTML in Markdown either, unfortunately). So I'm afraid I'm going leave it as is.